classifieds/classifieds-war/src/main/java/org/silverpeas/components/classifieds/control/ClassifiedsSessionController.java

```diff
@@ -365,13 +365,13 @@ public void deleteClassified(String classifiedId) {
    * @param isAdmin : boolean
    */
   public synchronized void updateClassified(ClassifiedDetail classified, boolean isUpdate,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "updateClassified" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)